package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Denormalizer for normalizr",
   "main": "lib/index.js",
   "files": [
+    "dist",
     "lib"
```

Comment:
We can use only one built version 😊 Remove `lib` and set `"main": "dist/index.js"` above.